packages/core/src/endowmentsToolkit.js

```diff
@@ -750,19 +848,6 @@ function defaultCreateFunctionWrapper(sourceValue, unwrapTest, unwrapTo) {
  * @returns {(...args: any[]) => any}
  */
 
-/**
- * Makes a copy of all globals from the global ref to a target and wraps them
- * with the wrapper this endowmentsToolkit was configured to use. It also copies
- * all circular references to the root package compartment globalThis.
- *
- * @callback CopyWrappedGlobals
```

Comment:
It was annoying me that these are at the bottom of the file, so I moved one to see if anything fails.
@boneskull do you remember why we moved them there? I'm not seeing anything failing yet.